scripts/roles/todd/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+  - include_vars: "{{ansible_os_family}}.yml"

```

Comment:
Where are the specific OS Family vars located to provide the service name handlers?